python/qemu/ot/otp/const.py

```diff
@@ -221,6 +228,22 @@ def get_digest_pair(self, name: str, prefix: str) -> dict[str, str]:
                 odict[oname] = values[idx]
         return odict
 
+    def get_scrambling_key(self, name: str) -> str:
+        """Get the value of a scrambling key.
+           :param name: One of the enumerated values, see #get_scrambling_keys
+        """
+        try:
+            idx = self._enums['key'][name]
+        except KeyError as exc:
+            raise ValueError(f'Unknown scrambling key {name}') from exc
+        try:
+            key_values = self._consts['key']
+        except KeyError as exc:
+            raise ValueError('No scrambling key constants found') from exc
+        if len(key_values) <= idx:
+            raise ValueError(f'No such key {name} in the key array') from exc
```

Comment:
Ah, I tried to push this change but you had already merged the PR 😅
```suggestion
            raise ValueError(f'No such key {name} in the key array')
```
It'll likely just throw an `UnboundLocalError` instead of a `ValueError` on this error case, so not a huge deal.

I'll open a PR to fix this.